### PR TITLE
Uploaded LamentinLosers

### PR DESCRIPTION
**Pull request by TheLocalPub**
Uploaded my first mission after some github headaches.

### CODE REVIEW

**Comment by TheLocalPub** [comment]
Ignore all the commits. I had to make file structure and stuff.

File: Missions/TheLocalPub/LamentinLosers.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+SCR_MissionHeader {
+ World "{E223CEC24B18A917}worlds/TheLocalPub/LamentinLosers/LamentinLosers.ent"
+ m_sName "COTVT45 Lamentin Losers"
```
Comment (joewhite94) [inline Missions/TheLocalPub/LamentinLosers.conf:3]: New naming convention per the readme is "TYPE (MIN-MAX) MissionName":

```suggestion
 m_sName "COTVT (2-45) Lamentin Losers"
```

Change 2 to whatever your minimum is

File: worlds/TheLocalPub/LamentinLosers [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (joewhite94) [inline worlds/TheLocalPub/LamentinLosers:1]: remove this file and create a folder called LamentinLosers within TheLocalPub to contain the mission

Comment (TheLocalPub) [reply worlds/TheLocalPub/LamentinLosers:1]: What? there is..

File: worlds/TheLocalPub/Prefabs/Characters/Factions/OPFOR/USSR_Army/Extra/Character_USSR_Crewman_driver_P.et [hunk 3]
```diff
@@ -0,0 +1,24 @@
+SCR_ChimeraCharacter : "{D2E883F5E6911DBA}Prefabs/Characters/Factions/OPFOR/USSR_Army/Character_USSR_Unarmed_P.et" {
```
Comment (joewhite94) [inline worlds/TheLocalPub/Prefabs/Characters/Factions/OPFOR/USSR_Army/Extra/Character_USSR_Crewman_driver_P.et:1]: We have renamed variants of the unarmed USSR crew in GC units if you wanted to use that version instead to cut out these prefabs, not a neccessity though

**Review by joewhite94 — APPROVED** [review]
looks good and works fine in local dedicated

**Pull request merged**